tests/test_shacl.py

```diff
@@ -0,0 +1,23 @@
+import pytest
+from pyshacl import validate
+
+
+@pytest.mark.parametrize(
+    ("shacl", "example"),
+    [
+        ("Catalog", "example-catalog"),
+        ("Dataset", "example-dataset"),
+        ("Distribution", "example-distribution"),
+    ],
+)
+def test_core_shapes(shacl, example):
+    shacl_graph = rf"Formalisation(shacl)/Core/PiecesShape/{shacl}.ttl"
```

Comment:
Can you move the paths out of this function ? I prefer to have additional functions getTest() and getShacl().

Reply:
Done